src/pages/Home/Home.module.scss

```
@@ -2,17 +2,16 @@
   align-items: center;
   display: flex;
   flex-direction: column;
-  gap: 1rem;
-  height: 100%;
-  justify-content: center;
+  height: calc(var(--homepage-layout-content-height));
+  justify-content: space-between;
 
   & .Title {
     font-size: 4rem;
     text-align: center;
   }
 
   & .AnimationContainer {
-    margin-bottom: -800px;
+    height: var(--animation-container-height);
```

could you try this instead, I think it's a simpler solution
```suggestion
    height: 45vh;
    margin-bottom: -50%;
```

and below in the media query

```css
 & .AnimationContainer {
      height: 30vh;
    }
```

you could also reduce the navbar and footer padding to `1rem` to get a bit more room.

---

Definitely simpler. I was unsure how much I should change. Thanks for the feedback

---

if you want all the way you could

in Navbar and Footer
- reduce the padding to `1rem`
- fix the height to `56px` in both

in `Home.module.scss`
- remove `justtify-content: center`

and finally in `Layout.module.scss`
```css
.Layout {
  align-items: center;
  background: linear-gradient(135deg, #211cab, #c74a95);
  background-size: cover;
  display: flex;
  flex-direction: column;
  /* add this line */
  height: 100vh;

  & .Content {
    flex: 1;
    max-width: 1344px;
    /* and this line */
    overflow: auto;
    padding: 3rem;
    width: 100%;
  }
}
```
this make only the content scrollable, leaving the navbar and footer always visible.